templates/agent/nodeattestor/nodeattestor_test.go

```diff
@@ -32,5 +35,19 @@ func Test(t *testing.T) {
 		},
 	})
 
-	// TODO: Invoke methods on the clients and assert the results
+	ctx := context.Background()
+
+	// TODO: Remove if no configuration is required.
+	_, err := configClient.Configure(ctx, &configv1.ConfigureRequest{
+		CoreConfiguration: &configv1.CoreConfiguration{TrustDomain: "example.org"},
+		HclConfiguration:  `{}`,
+	})
+	assert.NoError(t, err)
+
+	require.True(t, naClient.IsInitialized())
+	// TODO: Make assertions using the desired plugin behavior.
+	resp, err := naClient.AidAttestation(ctx)
```

Comment:
We should ensure the stream gets cleaned up properly using a cancelling context.
```suggestion
	ctx, cancel := context.WithCancel(ctx)
	defer cancel()
	stream, err := naClient.AidAttestation(ctx)
```

Reply:
[b1aedd0](https://github.com/spiffe/spire-plugin-sdk/pull/47/commits/b1aedd0ffadea2091feebbb031c9826014dd335d)